deploy/example_catalog/cr-app-tensorflow-notebook.json

```diff
@@ -2,36 +2,50 @@
     "apiVersion": "kubedirector.hpe.com/v1beta1",
     "kind": "KubeDirectorApp",
     "metadata": {
-        "name": "ml-jupyter-notebook",
-        "annotations": {
-            "token":"execute this command to get the authentication token 'kubectl exec <pod-name> jupyter notebook list' and use this token in Jupyter notebook."
-        }
+        "name" : "jupyter-notebook"
     },
```

Comment:
@saurabh-jogalekar This looks like your master branch and not MLOPs right ?

Reply:
@kmathur2 @Supekar
Yes, this is not MLOps image. Please refer to the latest commit for MLOps Apps.
Link: https://github.com/saurabh-jogalekar/kubedirector/commit/25319725d3af3491ea3c7e73c06ac203666f75f1

Reply:
this PR should be against MLOps branch

Reply:
Look like it is the correct branch but incorrect file name.

Reply:
This image is of the tensorflow+jupyter app that is currently used in HPECP. It is directly derived from tensorflow docker hub. The app we want the changes to go in would be [this](https://github.com/saurabh-jogalekar/kubedirector/blob/mlops-lift-and-shift/deploy/example_catalog/cr-app-tensorflowcpu.json)

Reply:
will re do the changes on the correct file